src/components/atoms/OcProgress/OcProgress.vue

```diff
@@ -1,14 +1,17 @@
 <template>
-  <progress
-    :value="value"
-    :max="max"
+  <div
```

Comment:
Adding a proper review later, but already pretty sure this misses `role="progressbar"` or sth of the likes

Reply:
👍